configuration/pih/htmlforms/clinicalHistory.xml

```diff
@@ -169,22 +169,100 @@
 
     <section sectionTag="fieldset" headerTag="legend" headerStyle="title" headerCode="Hereditarios">
 
-        <ifMode mode="VIEW" include="false" >
-                <script type="text/javascript">
-                    jq(function () {
-                        setupRespiratoryRisk('<lookup expression="fn.latestObs('PIH:12586').valueNumeric"/>',
-                                             '<lookup expression="fn.latestObs('PIH:12998').valueNumeric"/>'
-                        );
-                    });
-                </script>
-        </ifMode> 
-
         <p class="font-size-16 centered-header">Diagnósticos de antecedentes familiares:</p>
         <div class="four-columns margin-bottom">
-             <familyHistoryRelativeCheckboxes concept="CIEL:121375" label="Asma" relatives="PIH:FATHER,PIH:MOTHER,PIH:OTHER"/>
-             <familyHistoryRelativeCheckboxes concept="CIEL:119481" label="Diabetes" relatives="PIH:FATHER,PIH:MOTHER,PIH:OTHER"/>
-             <familyHistoryRelativeCheckboxes concept="CIEL:155" label="Epilepsia" relatives="PIH:FATHER,PIH:MOTHER,PIH:OTHER"/>
-             <familyHistoryRelativeCheckboxes concept="CIEL:139071" label="Enfermedad del corazón" relatives="PIH:FATHER,PIH:MOTHER,PIH:OTHER"/>
+            
+
+                <obsgroup groupingConceptId="CIEL:160593">
```

Comment:
Un `<obsgroup>` para cada enfermedad distinto. Queremos que podemos agrupar los valores correpsondientes. O, de otra perspectiva, no podemos tener mas de un `obs` con un particular `conceptId` al nivel de formulario. Si queremos repetir uno (como `PIH:RELATIONSHIPS OF CONTACT`) tiene que ser en `obsgroup`s distintos.